modules/currencies/currencies.model.js

```diff
@@ -0,0 +1,12 @@
+const currencyType = `    
+    _id: ID!
+    date: String!
```

Comment:
Might be a good idea to define Date type (custom scalar or by way of an existing package); String gives too much leeway and can get messy real quick

Reply:
fixed